cross/syncthing/Makefile

```diff
@@ -1,27 +1,42 @@
 PKG_NAME = syncthing
-PKG_VERS = 1.22.1
+PKG_VERS = 1.22.2
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)-source-v$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = https://github.com/syncthing/syncthing/releases/download/v$(PKG_VERS)
-PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
 EXTRACT_PATH = $(WORK_DIR)/src/github.com/$(PKG_NAME)
 PKG_DIR = $(PKG_NAME)
 
-DEPENDS = native/go
+BUILD_DEPENDS = native/go
+# add node for "next gen gui"
+BUILD_DEPENDS += native/nodejs
```

Comment:
Wow, is that really necessary?  I'd like to avoid blowing up the build complexity and time like that.

Reply:
@acolomb don't you agree that it is worth to include the next-gen-gui?
IMHO build time doesn't matter, only package size matters and this additional gui increases the package size less than 10% (< 1 MB).

Reply:
Originally the tech-ui used to be distributed as a single ZIP file, already built.  That should be dropped inside the `STGUIASSETS` dir, which is probably harder to specify in the DSM environment.  So making a path available for that is a good move, so people can drop the tech-ui or any other themes / modifications in there.  But I don't see much value in shipping the tech-gui by default, it's really more of an experiment and not well maintained.

At least not useful enough to justify the added build complexity.  E.g. on my machine, `--with-next-gen-gui` hangs somewhere in the process caused by some NPM failure I have absolutely no interest in debugging. And such a build doesn't work in my IDE's console because of NPM's colored and line-overwriting output. Many small details making life harder for simply building a Go package...

Reply:
Can we please get rid of this next-gen-gui stuff again?